### PR TITLE
Adds integration docs with SuperTokens

### PR DESCRIPTION
**Pull request by rishabhpoddar**
## What kind of change does this PR introduce?

Docs update

## Additional context

Adds docs about how to add SuperTokens auth, create a Supabase JWT, store that in a SuperTokens session, and query Supabase on the frontend and backend.

### CODE REVIEW

File: web/docs/guides/integrations/supertokens.mdx [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+![SuperTokens App Authenticated](/img/guides/integrations/supertokens/supabase_app_authenticated_screen.png)
+
+If you navigate to your table you should see a new row with the user's `user_id` and `email`.
+
+![Supabase Users table](/img/guides/integrations/supertokens/table_with_user.png)
+
+## Resources
+
+- [SuperTokens](https://supertokens.com/) official website.
+- [SuperTokens blog](https://supertokens.com/blog).
```
Comment (kiwicopple) [inline web/docs/guides/integrations/supertokens.mdx:398]:
```suggestion
```

I would like to remove the link to the blog - it doesn't directly help with the tutorial above

File: web/docs/guides/integrations/supertokens.mdx [hunk 2]
```diff
@@ -0,0 +1,402 @@
+---
+id: supertokens
+title: 'SuperTokens'
+description: "Create a Next.js application secured by Email-Password or Social Login from Stytch, and Supabase's Row Level Security (RLS)."
```
Comment (kiwicopple) [inline web/docs/guides/integrations/supertokens.mdx:4]:
```suggestion
description: "Create a Next.js application secured by SuperTokens and PostgreSQL Row Level Security."
```

**Review by kiwicopple — CHANGES_REQUESTED** [review]
a couple of suggestions 

Also would you mind running prettier on this one? Hopefully it will add line breaks after the Headdings

**Comment by rishabhpoddar** [comment]
> Also would you mind running prettier on this one? Hopefully it will add line breaks after the Headdings

Done! Thanks

**Comment by jscyo** [comment]
Hey @kiwicopple any update on the status of this PR being merged?

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**